src/main/java/com/example/nlearn/controllers/StudentsHomeController.java

```diff
@@ -0,0 +1,25 @@
+package com.example.nlearn.controllers;
+
+import com.example.nlearn.models.CustomOAuth2User;
+import com.example.nlearn.models.Test;
+import com.example.nlearn.services.TestService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("studentsHome")
+public class StudentsHomeController {
+
+    @Autowired
+    private TestService testService;
+
+    @GetMapping("/first5")
```

Comment:
remove mapping route or rename it to be without a number in the name

Reply:
done